loops-arrays-objects.js

```diff
@@ -31,9 +32,19 @@ const scores = [
 // A
 // D
 // ==========================================
-
-
-
+for (let i = 0; i < scores.length; i++) {
+    if (scores[i]["score"] < 60) {
```

Comment:
De implementatie om scores om te zetten in lettercijfers is correct. De if-else structuur is logisch en goed opgebouwd.